.coafile

```diff
@@ -0,0 +1,3 @@
+files = .coafile
+use_spaces=true
+bears = SpaceConsistencyBear 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/.coafile
+++ b/.coafile
@@ -1,3 +1,3 @@
 files = .coafile
 use_spaces=true
-bears = SpaceConsistencyBear 
+bears = SpaceConsistencyBear
```